Formula/arkade.rb

```diff
@@ -0,0 +1,31 @@
+class Arkade < Formula
+  desc "Open Source Kubernetes Marketplace"
+  homepage "https://blog.alexellis.io/kubernetes-marketplace-two-year-update/"
+  url "https://github.com/alexellis/arkade.git",
+      tag:      "0.8.11",
+      revision: "f6c134889f89c4bfe1b06f0f97037f16caf2b6a3"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/alexellis/arkade/cmd.Version=#{version}
+      -X github.com/alexellis/arkade/cmd.GitCommit=#{Utils.git_head}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    bin.install_symlink "arkade" => "ark"
+
+    (zsh_completion/"_arkade").write Utils.safe_popen_read("#{bin}/arkade", "completion", "zsh")
+    (bash_completion/"arkade").write Utils.safe_popen_read("#{bin}/arkade", "completion", "bash")
```

Comment:
let's also add completions for `fish`
```suggestion
    (zsh_completion/"_arkade").write Utils.safe_popen_read(bin"arkade", "completion", "zsh")
    (bash_completion/"arkade").write Utils.safe_popen_read(bin/"arkade", "completion", "bash")
    (fish_completion/"arkade.fish").write Utils.safe_popen_read(bin/"arkade", "completion", "fish")
```

Reply:
I would have added them in the first place, but believed this command output that fish is not supported: 
```bash
$ arkade completion
Use "arkade completion SHELL" to generate SHELL completion for:
    - bash
    - zsh
```
fish works too, though 😄